app/Http/Controllers/Backend/TokenController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
```

Comment:
Can you move this to a subnamespace like `App\Http\Controllers\Backend\User` for example?